code/modules/admin/topic.dm

```diff
@@ -794,16 +808,17 @@
 						return 1
 				if("Cancel")
 					return
+			return
 
 		//Unbanning job list
 		//all jobs in job list are banned already OR we didn't give a reason (implying they shouldn't be banned)
-		if(LAZYLEN(SSjobs.titles_to_datums)) //at least 1 banned job exists in job list so we have stuff to unban.
+		if(length(bannedlist))
```

Comment:
```suggestion
		if(LAZYLEN(bannedlist))
```
LAZYLEN is currently macro'd out to length() anyway, but it would save on maintenance if Bay ever edits the definition. Plus, standardization is good.